packages/@o3r/eslint-config/eslint.local.config.mjs

```diff
@@ -17,12 +17,24 @@ export default [
       parserOptions: {
         tsconfigRootDir: __dirname,
         project: [
-          'tsconfig.build.json',
           'tsconfig.builders.json',
           'tsconfig.eslint.json',
           'tsconfig.spec.json'
         ]
       }
     }
+  },
+  {
+    name: '@o3r/eslint-config/projects/commonjs',
+    files: ['**/*.cjs'],
+    languageOptions: {
+      sourceType: 'commonjs',
```

Comment:
this is potentially global at workspace level (for `.cjs`).
A similar rule can be set for `.mjs`

Reply:
A global rule is set, here it is needed to linked the tsconfig.build